src/SWP/Bundle/FixturesBundle/DataFixtures/ORM/LoadWidgetsData.php

```diff
@@ -26,11 +28,14 @@ class LoadWidgetsData extends AbstractFixture implements FixtureInterface, Order
      */
     public function load(ObjectManager $manager)
     {
+        /** @var TenantInterface $tenant */
```

Comment:
I'm surprised this works. Suggested way of sharing objects between fixtures is to add a reference. See section with tittle "Sharing Objects between fixtures" here: http://symfony.com/doc/current/bundles/DoctrineFixturesBundle/index.html